bin/internal/shared.bat

```diff
@@ -131,17 +132,17 @@ GOTO :after_subroutine
       SET /A total_tries=10
       SET /A remaining_tries=%total_tries%-1
       :retry_pub_upgrade
-        ECHO Running pub upgrade...
+        ECHO Running pub upgrade... 1>&2
         CALL "%pub%" upgrade "%VERBOSITY%" --no-precompile
         IF "%ERRORLEVEL%" EQU "0" goto :upgrade_succeeded
-        ECHO Error (%ERRORLEVEL%): Unable to 'pub upgrade' flutter tool. Retrying in five seconds... (%remaining_tries% tries left)
+        ECHO Error (%ERRORLEVEL%): Unable to 'pub upgrade' flutter tool. Retrying in five seconds... (%remaining_tries% tries left) 1>&2
         timeout /t 5 /nobreak 2>NUL
         SET /A remaining_tries-=1
         IF "%remaining_tries%" EQU "0" GOTO upgrade_retries_exhausted
         GOTO :retry_pub_upgrade
       :upgrade_retries_exhausted
         SET exit_code=%ERRORLEVEL%
-        ECHO Error: 'pub upgrade' still failing after %total_tries% tries, giving up.
+        ECHO Error: 'pub upgrade' still failing after %total_tries% tries. 1>&2
```

Comment:
If I don't shorten this message, then `:upgrade_succeeded` cannot be located from the `GOTO` on L137 when running the hostonly windoows machines.

Reply:
good question!

I've seen some information that GOTO label can behave weird if the label is outside of some certain size block, but from what I read that only applies if unix line endings are used. however, without fail the hostonly windows machines could not locate the `upgrade_succeeded` label with the longer message.